source/stable-release_iot/feature-delta-iot.rst

```diff
@@ -0,0 +1,76 @@
+.. _feature-delta-iot:
+
+Feature Delta Offered by Celadon IoT
+####################################
+
+Following are the additional features provided by Celadon IoT.
+
+.. contents::
+    :depth: 1
+    :local:
+* 100% Google Conformance on Comet Lake(CML),Tiger Lake(TGL) & Elkhart Lake(EHL) with Production ready
```

Comment:
Production ready - there was an action to confirm if you can claim this with SDL completion. Is that done now?

Reply:
SDL Completion is Done